src/cbmc/cbmc_solvers.cpp

```diff
@@ -164,8 +166,10 @@ std::unique_ptr<cbmc_solverst::solvert> cbmc_solverst::get_smt2(
   {
     if(solver==smt2_dect::solvert::GENERIC)
     {
-      error() << "please use --outfile" << eom;
-      throw 0;
+      throw invalid_user_input_exceptiont(
+        "required filename not provided",
+        "--outfile",
+        "provide a filename with --outfile");
```

Comment:
Consistently use uppercase or lowercase for those messages (it's all uppercase in src/cbmc/bmc.cpp).

Reply:
I changed the ones in `src/cbmc/bmc.cpp` to use lowercase, to make it more consistent with the rest of the error messages across CBMC.